Design/Workbench/design-layout.md

```diff
@@ -0,0 +1,149 @@
+Summary
+=======
+
+This document describes the layout and high-level details of the new matplotlib-based workbench that will replace MantidPlot.
+
+Workflow
+========
+
+The new workbench will be developed on a long-running feature branch within the main [`mantid`][mantidrepo] repository. The exception to this is the new directory
+structure described below for the graphical components. These changes will be made on the `master` branch prior to any work starting on the workbench. This
+will minimize the effect of merge conflicts on existing components that are going to be reused.
```

Comment:
I think this might be a bit optimistic. I would probably say at least 2-3 days to shuffle and get everything compiling. There's a fair bit of file shuffling and changes to things like include paths to deal with.